overlay/files/usr/lib/os-release.tmpl

```diff
@@ -1,14 +1,13 @@
-NAME="${OS_NAME}"
-VERSION="${OS_VERSION}"
-ID="${OS_ID}"
-ID_LIKE="${OS_NAME}"
-VERSION_ID="${OS_VERSION}"
-PRETTY_NAME="${OS_NAME} ${OS_VERSION}"
-ANSI_COLOR="0;32"
-BUG_REPORT_URL="${BUG_REPORT_URL}"
-HOME_URL="${HOME_URL}"
-IMAGE_REPO="${OS_REPO}"
-IMAGE_LABEL="${OS_LABEL}"
-GITHUB_REPO="${GITHUB_REPO}"
-VARIANT="${VARIANT}"
-FLAVOR="${FLAVOR}"
\ No newline at end of file
+KAIROS_NAME="${OS_NAME}"
```

Comment:
@mudler  Would it make sense to prefix all variables with KAIROS? Seems to be the recommendation, but maybe I'm missing something

> The file is extensible? Awesome! I want a new field XYZ= in it! Sure, it's extensible, and we are happy if distributions extend it. Please prefix your keys with your distribution's name however. Or even better: talk to us and we might be able update the documentation and make your field standard, if you convince us that it makes sense.

http://0pointer.de/blog/projects/os-release

Reply:
yep sounds about right. but we should be careful, as I remind we read from those in several places down the line, for instance to get the current flavor  https://github.com/kairos-io/kairos-sdk/blob/77ed7ba29815f03090ad63bf0ec3e56dfa83a229/utils/utils.go#L98